library/build.gradle

```diff
@@ -62,7 +62,6 @@ android {
         all {
             it.java.srcDir "src/$it.name/kotlin"
         }
-        main.java.srcDirs += 'src/main/kotlinX'
```

Comment:
Why extension has been removed from the library?

Reply:
I moved them into the main source set from the separate `kotlinX` source set, simplifies the project. It was two files and ~10 lines in their own source set.

Reply:
The idea with that is to have a common module where all "Third-party class extensions" live, so whenever anyone needs to check if an extension exists or create a new one, he/she knows where to go.
On the other hand, using the same package name of the class over which the extension acts, the imports are simpler and they start to make sense

Reply:
I've never seen these forced into the package of the original types, always had them packaged under the project's own package name. After all, it's part of our codebase. Importing them from packages like `java.util` can only make things more confusing for developers. Discoverability is the same either way, and in general extensions should be found through IDE tooling, e.g. autocompletion.

It's a good idea to have them in a dedicated place so that we can look them up, how about either a `com.getstream.sdk.chat.extensions` package, or maybe under the utils, so in `com.getstream.sdk.chat.utils.extensions`? In there, we can have files named after the types that the extensions are for, like `Date.kt` or `When.kt`.